src/routes/resource-edit-route/resource-edit-route.test.js

```diff
@@ -0,0 +1,333 @@
+import { MemoryRouter } from 'react-router-dom';
+
+import {
+  render,
+  cleanup,
+  fireEvent,
+} from '@testing-library/react';
+
+import ResourceEditRoute from './resource-edit-route';
+import Harness from '../../../test/jest/helpers/harness';
+
+const history = {
+  replace: jest.fn(),
+};
+
+const location = {
+  pathname: 'pathname',
+  search: '',
+  hash: '',
+};
+
+const provider = {
+  id: 'providerid',
+  providerToken: {
+    value: 'provider-token',
+    prompt: '',
+  },
+  data: {
+    isLoaded: true,
+    isLoading: false,
+  },
+  proxy: {
+    id: 'proxy-id',
+  },
+};
+
+const id = `${provider.id}-titleid`;
+
+const match = {
+  isExact: true,
+  params: {
+    id,
+  },
```

Comment:
It could be written in one line
```suggestion
  params: { id },
```

Reply:
You are right, thanks!